src/GraphQL.Tests/BasicQueryTestBase.cs

```diff
@@ -34,8 +34,8 @@ public ExecutionResult AssertQuery(Action<ExecutionOptions> options, ExecutionRe
     {
         var runResult = Executer.ExecuteAsync(options).Result;
 
-        var writtenResult = Writer.Serialize(runResult);
-        var expectedResult = Writer.Serialize(expectedExecutionResult);
+        string writtenResult = Writer.Serialize(runResult);
```

Comment:
I do not think these suggestions should be implemented

Reply:
https://github.com/graphql-dotnet/graphql-dotnet/blob/ac4f9c94e79163036fad38510b2829de4e800f1b/.editorconfig#L87